src/com/project/graph/Edge.java

```diff
@@ -0,0 +1,49 @@
+package com.project.graph;
+
+public class Edge {
+	private Node from;
+	private Node to;
+	private double speedLimit;
+	
+	public Edge(Node from, Node to, double speedLimit){
+		this.from = from;
+		this.to = to;
+		this.speedLimit = speedLimit;
+	}
+	
+	public Node getFrom(){
+		return from;
+	}
+	
+	public void setFrom(Node newFrom){
+		from = newFrom;
+	}
+	
+	public Node getTo(){
+		return to;
+	}
+	
+	public void setTo(Node newTo){
+		to = newTo;
+	}
+	
+	public double getSpeedLimit(){
+		return speedLimit;
+	}
+	
+	public void setSpeedLimit(double newSpeedLimit){
+		speedLimit = newSpeedLimit;
+	}
+	
+	public double getLength(){
```

Comment:
It's the length of the road. Since crossings are represented by co-ordinates in the plane, and we're assuming that all roads are straight, the length of an edge is the distance between the vertices it connects.